src/patches/widescreen.cpp

```diff
@@ -581,10 +581,11 @@ SMS_PATCH_BL(SMS_PORT_REGION(0x8013F430, 0x80133FAC, 0, 0), patchLevelSelectPosi
 
 static void scaleUnderWaterMask(Mtx mtx, f32 x, f32 y, f32 z) {
     CPolarSubCamera *camera = gpCamera;
-    x *= camera->mProjectionFovy / 50.0f;
-    x *= getScreenToFullScreenRatio();
+    const f32 fovtangent = tan(angleToRadians(camera->mProjectionFovy * 0.5f)) * 2.0;
```

Comment:
Super Mario Sunshine has `tanf` for float precision, rather than the double precision `tan`. Let's use that instead.